app/views/planning_applications/assessment/meetings/index.html.erb

```diff
@@ -0,0 +1,41 @@
+<% content_for :page_title do %>
+  Meetings - <%= t("page_title") %>
+<% end %>
+
+<%= render(
+      partial: "shared/proposal_header",
+      locals: {heading: "View meetings"}
+    ) %>
+
+<div class=govuk-width-container>
+  <% if @meetings.any? %>
+    <table class="govuk-table">
+      <caption class="govuk-table__caption govuk-table__caption--m">Meeting history</caption>
+      <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th scope="col" class="govuk-table__header">Meeting date</th>
+          <th scope="col" class="govuk-table__header">Case officer</th>
+          <th scope="col" class="govuk-table__header">Notes (optional)</th>
+        </tr>
+      </thead>
+      <% @meetings.each do |meeting| %>
+        <% if meeting.occurred_at %>
```

Comment:
Should this be `meeting.persisted?`
This is so we don't show the new unsaved record in the table right.